platform-hub-api/spec/controllers/concerns/authentication_spec.rb

```diff
@@ -13,8 +13,8 @@ def index
 
   before do
     expect(@controller).to receive(:authenticated?).and_call_original
-    expect(@controller).to receive(:current_user).and_call_original
-    expect(@controller).to receive(:token_and_options).and_return([auth_token, nil])
+    allow(@controller).to receive(:current_user).and_call_original
+    allow(@controller).to receive(:token_and_options).and_return([auth_token, nil])
```

Comment:
Any particular reason to change these? Now they won't be explicitly checked as part of the spec.